docs/building-a-node-with-ldk/setting-up-a-channel-manager.md

```diff
@@ -0,0 +1,1276 @@
+# Setting up a ChannelManager
+
+The ChannelManager is responsible for several tasks related to managing channel state. This includes keeping track of many channels, sending messages to appropriate channels, creating channels and more.
+
+## Adding a ChannelManager
+
+To add a `ChannelManager` to your application, run:
+
+<CodeSwitcher :languages="{rust:'Rust', kotlin:'Kotlin', swift:'Swift'}">
+  <template v-slot:rust>
+
+```rust
+use lightning::ln::channelmanager;
+
+let channel_manager = ChannelManager::new(
+  &fee_estimator,
+  &chain_monitor,
+  &broadcaster,
+  &router,
```

Comment:
The most recent version of LDK has `ChannelManager` take `Router`, so this is correct. But a later section with a similar examples is using an older version of LDK that does not take a `Router`.